scripts/valid-links.py

```
@@ -0,0 +1,165 @@
+import re
+import requests
+import json
+from pathlib import Path
+from urllib.parse import urlparse
```

Ruff (0.8.2)

`urllib.parse.urlparse` imported but unused

Remove unused import: `urllib.parse.urlparse`

(F401)